app/models/bot.rb

```diff
@@ -12,5 +12,8 @@ class Bot < ActiveRecord::Base
   validates :hair_color, presence: true
   validates :user_id, presence: true
 
+  def avatar
+   gender + '-' + hair_color + '-' + eye_color + '.png'
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.